### PR TITLE
add styling to landing

### LINKED ISSUE #5: Improve landing page

### LINKED ISSUE #7: modify the feedback scale
scale size

### PR DESCRIPTION
**Pull request by PyNate**
closes #5

### CODE REVIEW

File: client/app/auth/landing.html [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
-		<div class="landing-heading">
-	    <p class="greeting">Welcome</p>
+	<div class="landing-heading">
+    <div class="row>">
+	    <p class="greeting">Welcome to</p>
 			<h1 id="spkr">SPKR</h1>
+      <p class="greeting">an app to improve your public speaking skills</p>
 		</div>
 	<div class="row">
 	    <a type="button" class="btn btn-danger btn-lg" href="/#/login">Login</a>
```
Comment (kylebakerio) [inline client/app/auth/landing.html]: Please fix the indentation.

**Comment by PyNate** [comment]
Indentation fixed

**Comment by kylebakerio** [comment]
Hmm.... Still doesn't look fixed here. Not sure what's going on. Might try pushing your changes, or confirming that you didn't forget to stage that file before you committed, perhaps? If neither of those,  let's try to figure out what's going on in person.

**Comment by PyNate** [comment]
Check it now. I deleted all of the lines and recommitted them.

**Pull request merged**